LibNFS.xs

```diff
@@ -974,7 +974,12 @@ static void _set_unix_authn(pTHX_ struct nfs_context* nfs, SV* value_sv) {
         SV** svp = av_fetch(authn_av, n, 0);
         assert(svp);
 
-        nums[n] = exs_SvUV(*svp);
+        UV value = exs_SvUV(*svp);
+        if (value > UINT32_MAX) {
```

Comment:
It’s not obvious why this needs to be so, esp since upstream uses int, not uint32_t.

I’ll add an appropriate comment.